.github/workflows/run-tests-on-new-version-of-ddc-sc.yml

```diff
@@ -0,0 +1,39 @@
+name: Run tests with latest version of DDC Smart Contract
+on:
+  repository_dispatch:
+    types: DDC_SMART_CONTRACT_VERSION_TRIGGER
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Maximize build space
+        uses: easimon/maximize-build-space@v3
+        with:
+          root-reserve-mb: 512
+          swap-size-mb: 1024
+          remove-dotnet: 'true'
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Checkout submodules
+        run: git submodule update --init --recursive
+      - name: Free space
```

Comment:
Why do we need to check the free space before build?

Reply:
I don't know, it is already in other workflows. If somebody add it, i think it is really needed.